dockerfiles/init/modules/che/templates/che.env.erb

```diff
@@ -23,7 +22,8 @@ JPDA_ADDRESS=<%= scope.lookupvar('che::che_debug_port') %>
 
 CHE_DATABASE=/data
 CHE_WORKSPACE_STORAGE=/data/workspaces
-CHE_WORKSPACE_LOGS=/data/logs/machine
+CHE_LOGS_DIR=/logs
```

Comment:
@garagatyi yes correct, this is the fix. Before it was hardcoded in che-server docker image. Now it is set only if not provided. And cli is providing the property so it handle it correctly.